### ISSUE TITLE
Proofs Note

### ISSUE THREAD

**Issue by nadalin** [w3c/vc-data-model#612]
"No single proof mechanism is currently standardized or recommended for verifiable credentials." This is incorrect as JWS is standardized in IETF and widely used. Suggest rewording this NOTE

**Comment by TallTed** [comment]
The operative elements in your quoted sentence are "No **single** ... **for verifiable credentials**."  In other words, we're not picking a winner.

That said, a viable rewording of this sentence might be `As discussed in Section <a href="#conformance"></a>, there are multiple viable proof mechanisms, and this specification does not standardize nor recommend any single proof mechanism for use with <a>verifiable credentials</a>.`  What do you think?

**Comment by nadalin** [comment]
+1

**Comment by burnburn** [comment]
WG resolution on 28 May 2019 call:

RESOLUTION: Issue #612 is addressed via PR #614, which rewords the text of concern in a non-normative way. Issue #612 should be closed after 7 days from when the PR was merged.

**Comment by nadalin** [comment]
So no real interop can take place based upon PR #614

**Comment by burnburn** [comment]
Your interop concern is being tracked by issue #632.  Closing.
